gspread/auth.py

```diff
@@ -138,14 +142,24 @@ def oauth(scopes=DEFAULT_SCOPES, flow=local_server_flow):
         sh = gc.open("A spreadsheet")
         sh.sheet1.update('A1', '42')   # <-- this will not work
 
+    If you're storing your user credentials in a place other than the
+    default, you may provide a path to that file like so::
+
+        gc = gspread.oauth(credentials_filename='/alternative/path/authorized_user.json')
```

Comment:
The parameter name in the docs should match the new parameter name in the function.

Reply:
done